### PR TITLE
Make SvgData size methods public

### PR DESCRIPTION
**Pull request by rosehuds**
I found them useful 🤷

also moved the private method downwards to preserve "public methods at the top"

### CODE REVIEW

File: druid/src/widget/svg.rs [hunk 1]
```diff
@@ -144,22 +144,8 @@ impl SvgData {
...
-        // TODO respect preserveAspectRatio
-        let t = Affine::translate((viewbox.min_x(), viewbox.min_y()));
-        let scale =
-            Affine::scale_non_uniform(size.width / viewbox.width(), size.height / viewbox.height());
-        scale * t
-    }
-
     /// Get the viewbox for the svg. This is the area that should be drawn.
-    fn viewbox(&self) -> Rect {
+    pub fn viewbox(&self) -> Rect {
```
Comment (cmyr) [inline druid/src/widget/svg.rs:148]: I would call these methods `svg_viewbox` and `svg_size` respectively, and clarify in the docs that these are the sizes reported for the underlying svg image, not for the widget.

**Review by cmyr — APPROVED** [review]
Thanks, this seems useful.

A suggestion inline, but otherwise I'm happy to see this merged :)

File: druid/src/widget/svg.rs (hunk 1 above)
Comment (rosehuds) [reply druid/src/widget/svg.rs:148]: I should clarify this - `SvgData` seems to me to only deal with literally the data of the SVG it was created from, so it is fairly clear that "size" refers to the SVG - the type serves no other purpose. All widget interactions are done by `Svg`.

Comment (cmyr) [reply druid/src/widget/svg.rs:148]: thanks for the clarification!

**Pull request merged**